docs/js/react.js

```diff
@@ -1109,6 +1109,9 @@ var ReactClass = {
       this.props = props;
       this.context = context;
       this.refs = emptyObject;
+      if (this.updater && "development" !== 'production') {
```

Comment:
Here I'm following the example of lines 1100 - 1102:

```
if ("development" !== 'production') {
         "development" !== 'production' ? warning(this instanceof Constructor, 'blahblah') : void 0;
       }
```

I must admit that I'm not quite sure how this logic works, how will `"development" !== 'production'` not end up being true?